Example/Agrume Example/MultipleImagesCollectionViewController.swift

```diff
@@ -39,6 +39,14 @@ final class MultipleImagesCollectionViewController: UICollectionViewController {
     agrume.didScroll = { [unowned self] index in
       self.collectionView?.scrollToItem(at: IndexPath(item: index, section: 0), at: [], animated: false)
     }
+    agrume.onLongPress = agrume.savePhotoOnLongPress
+    agrume.photoSavedToLibrary = { error in
+      if error != nil { // we got back an error!
+        print"Could not save your photo")
```

Comment:
This doesn't compile. Missing opening parentheses

nit: even if it's _just_ sample code, this is better expressed as a `guard` statement